### ISSUE TITLE
docs: Create Pull Request template

### ISSUE THREAD

**Issue by annabauza** [anitab-org/stem-diverse-tv-cms#6]
## Describe the bug/feature
There is no pull request template for this repository as of now. Therefore the pull request template mentioned below needs to be created and placed in `./github/PULL_REQUEST_TEMPLATE.md` (so that it gets loaded).

Template attached below: 

```
### Description

Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes # [ISSUE]

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bugfix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.


### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] I have written Docs whenever is applicable


**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules
```

## Expected behavior
- [ ] Created a ./github/PULL_REQUEST_TEMPLATE.md
- [ ] Pull request template is preloaded when contributor clicks new pull request

*Expected time to complete: 1 hour*

**Comment by chinmaym07** [comment]
@annabauza I want to take this up.

**Comment by annabauza** [comment]
@chinmaym07 yours

**Comment by Aaishpra** [comment]
@annabauza this issue is already fixed by #11 can you please close this?
Thanks
